TrigonGithubHanuka2018/src/org/usfirst/frc/team5990/robot/commands/CollectorOut.java

```diff
@@ -0,0 +1,47 @@
+package org.usfirst.frc.team5990.robot.commands;
+
+import org.usfirst.frc.team5990.robot.Robot;
+import org.usfirst.frc.team5990.robot.RobotMap;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class CollectorOut extends Command {
+	
+	private double power;
+	
+    public CollectorOut(double power) {
+        // Use requires() here to declare subsystem dependencies
```

Comment:
This is the exact same command as CollectIn!
they both get a power param
delete